packages/client/src/workflow-client.ts

```diff
@@ -829,6 +1026,28 @@ export class WorkflowClient extends BaseClient {
     runIdForResult,
     ...resultOptions
   }: WorkflowHandleOptions): WorkflowHandle<T> {
+    // TODO (dan): Convert to class with this as a protected method
+    const _startUpdate = async <Ret, Args extends unknown[]>(
```

Comment:
I vote for leaving it as is in this PR since having it invisible in the closures is actually  slightly better DX. Maybe we can find an as-good solution when we move to a class.